docs/robots/ridgeback/integration_ridgeback.mdx

```diff
@@ -1,12 +1,161 @@
-:::warning todo—under construction
+To attach custom hardware to Ridgeback, you will have to take care of mechanical mounting, electrical
+supply, and software integration. This guide aims to equip you with respect to these challenges.
+
+---
+
+### Mechanical Mounting
+
+When determining mechanical mounting, you can use the Standard mounting pattern on the Ridgeback.
+At this time, the "PACS" mounting system is not available for Ridgeback.
+
+#### Mechanical, Standard {#ridgeback-payload-mechanical-integration}
+
+The payload integration plate can be used to mount external payloads on top of the Ridgeback. The plate
+is made of aluminum, which allows Ridgeback to support payloads up to 100 kg (220 lbs). Ridgeback's
+batteries are positioned low in the chassis and slightly rearward of center of the robot to balance the
+weight distribution when mounting front-facing manipulator payloads. To minimize the possibility of tipping
+over, payload structures should always be mounted as close to center as possible.
+
+##### Payload Mounting Holes
+
+Located at the front-end of the mounting plate are two 5/8"-11 screw holes for mounting Baxter, UR5/UR510 manipulator
+arms, or any other payload structure. These holes are indicated in the figure below. If you purchased the Baxter
+or UR5/UR10 package from Clearpath Robotics, Ridgeback will come with the required hardware adapters to securely mount
+the payload structures to the plate.
+
+If your payload structure requires additional mounting holes or a different hole configuration, the plate can be removed
+and additional holes can be drilled. To remove the mounting plate, simply remove the screws that form the diamond pattern
+on top of the plate.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/payload_integration_plate.png"
+      width="600"
+    />
+    <figcaption>Payload integration plate</figcaption>
+  </figure>
+</center>
+
+:::warning
+
+Permanent damage resulting from custom modifications to the mounting plate is not covered under warranty and may not be
+supported by Clearpath Support. Please contact our support team if you require assistance or have any questions relating
+to custom modifications.
+
+:::
+
+### Electrical Integration {#ridgeback-payload-electrical-integration}
+
+Except for bus-powered USB cameras, most payloads have separate leads for power and data.
+
+#### Data Connections
+
+The two larger holes on the left and right side of the payload integration plate allow you to pass
+electrical wires and cables from the mounted payloads into the User Bay. Electrical wires should always
+pass through the provided plastic grommets to protect against cutting and abrasion.
+
+Ridgeback's internal computer options support USB3 and Ethernet connectivity. Where available, the
+PCIe slot may be used to supply Firewire, Thunderbolt, or additional USB3 ports, as necessary.
+
+Additionally, the User Bay may be used for an Ethernet switch, when attaching multiple
```

Comment:
Consider changing this to:

'An Ethernet switch or PoE power injector can be added to the User Bay as required.'

Reply:
The PCIe slot can't be used for anything because of the metal cover over the computer. That whole line should be removed.

Reply:
Updated the line that Rhys suggested. Removed the line that Jeff suggested.